forms-flow-web/src/components/Form/Item/Edit.js

```diff
@@ -449,377 +249,264 @@ const Edit = React.memo(() => {
-                          </div>
-                        </div>
+                </Card.Body>
+              </Card>
+            </div>
+            <div className={`wraper flow-wraper ${showFlow ? 'visible' : ''}`}>
+              <Card>
+                <Card.Header>
+                  <div className="d-flex justify-content-between align-items-center" style={{ width: "100%" }}>
+                    <div className="d-flex align-items-center justify-content-between">
+                      <div className="mx-2 builder-header-text">Flow</div>
+                      <div>
+                        <CustomButton
+                          variant="secondary"
+                          size="md"
+                          icon={<HistoryIcon />}
+                          label={<Translation>{(t) => t("History")}</Translation>}
+                          onClick={handleHistory}
+                          dataTestid={"cancelBtndataTestid"}
+                          ariaLabel={"cancelBtnariaLabel"}
+                        />
+                        <CustomButton
+                          variant="secondary"
+                          size="md"
+                          className="mx-2"
+                          label={<Translation>{(t) => t("Preview & Variables")}</Translation>}
+                          onClick={handlePreviewAndVariables}
+                          dataTestid={"cancelBtndataTestid"}
+                          ariaLabel={"cancelBtnariaLabel"}
+                        />
+                        {/* <CustomButton
+                          variant="secondary"
+                          size="md"
+                          className="mx-2"
+                          label={<Translation>{(t) => t("Switch to BPMN")}</Translation>}
+                          onClick={setWorkflowType}
+                          dataTestid={"cancelBtndataTestid"}
+                          ariaLabel={"cancelBtnariaLabel"}
+                        /> */}
                       </div>
                     </div>
-                  </Collapse>
-                </div>
-              </div>
-
+                    <div>
+                      <CustomButton
+                        variant="primary"
+                        size="md"
+                        className="mx-2"
+                        label={<Translation>{(t) => t("Save Flow")}</Translation>}
+                        onClick={saveFlow}
+                        dataTestid={"save-form-layout"}
+                        ariaLabel={"Save Form Layout"}
+                      />
+                      <CustomButton
+                        variant="secondary"
+                        size="md"
+                        label={<Translation>{(t) => t("Discard Changes")}</Translation>}
+                        onClick={discardChanges}
+                        dataTestid={"cancelBtndataTestid"}
+                        ariaLabel={"cancelBtnariaLabel"}
+                      />
```

Comment:
please follow kebab casing while giving datattestid
give a meaningfull name for aria label